sdk/core/azure-core-amqp/inc/azure/core/amqp/internal/connection.hpp

```diff
@@ -3,12 +3,12 @@
 
 #pragma once
 
-#include "azure/core/amqp/models/amqp_protocol.hpp"
 #include "common/async_operation_queue.hpp"
 #include "connection_string_credential.hpp"
-#include "models/amqp_value.hpp"
 #include "session.hpp"
 
+#include <azure/core/amqp/internal/models/amqp_protocol.hpp>
+#include <azure/core/amqp/models/amqp_value.hpp>
```

Comment:
For good or for bad, we currently include headers from the same library via quotes, and all other libraries via angle brackets.
This only includes `inc` and `src`, and does not apply to samples.

Reply:
The "" vs <> is done to prioritize our headers over system headers.  Its behavior that is compiler specific, but if a compiler supports it, we benefit from it.

Using `""` prioritizes headers in the current working directory over system headers. `<>` is used for system headers or other packages.

Reply:
In this case the paths are NOT relative to the current working directory, so the prioritization isn't relevant. These are absolute paths relative to a -I directive.

That means that they *are* effectively system headers, and I treated them as such.

Having said that, I don't care one way or another on this one.

Reply:
I would assume this could matter if you have a system-wide azure SDK library installed, and at the same time you develop a library, and if you use `<>`, it would find a header from a system-wide installation, while `""` could find the header from your development location.

Reply:
If you have a system wide azure sdk library which has include file names who match the names of your package, you have a different problem.

And if the path is not relative to your current directory, `""` behaves the same as `<>` so it won't help.